lib/buildingsync/model_articulation/building_subsection.rb

```diff
@@ -59,13 +56,15 @@ def read_xml(subsection_element, standard_template, occ_type, bldg_total_floor_a
       # based on the occupancy type set building type, system type and bar division method
       read_bldg_system_type_based_on_occupancy_type(subsection_element, occ_type, ns)
 
-      @space_types = get_space_types_from_building_type(@bldg_type, standard_template, true)
-
       @subsection_element = subsection_element
 
       # Make the standard applier
-      @standard = Standard.build("#{standard_template}_#{@bldg_type}")
-      OpenStudio.logFree(OpenStudio::Info, 'BuildingSync.BuildingSubsection.read_xml', "Building Standard with template: #{standard_template}_#{@bldg_type}")
+      begin
+        $open_studio_standards = Standard.build("#{standard_template}_#{@bldg_type}")
```

Comment:
I don't really like seeing the global variable here... is there a reason it has to be global?

Reply:
remove the global variable and refactored the code to better deal with this open_studio_standard generation and use at the building level